src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/opentelemetry/proto/agent/README.md

```diff
@@ -1,10 +0,0 @@
-# OpenTelemetry Agent Proto
```

Comment:
why remove?

Reply:
This is just describing the proto files, there is already one README.md on `Implementation/` dir that points to the original repo. Anyway, I will try to explore some ways to not have the proto sources directly here. I will create a separate issue for that.